src/core/config.ts

```diff
@@ -53,14 +54,11 @@ async function loadConfig<TConfigType>(
     return config;
 }
 
-export async function loadPlatformConfig(
-    ctx: Context
-): Promise<PlatformConfig> {
+export async function loadPlatformAndKeysConfig(url: string): Promise<PlatformConfigAndKeys> {
     return (
-        (await loadConfig<PlatformConfig>(
-            `${ctx.urlForService(Service.PUSH)}/v1/web/config`,
-            ConfigCacheType.PLATFORM,
-            ctx
+        (await loadConfig<PlatformConfigAndKeys>(
+            url,
+            ConfigCacheType.PLATFORM
```

Comment:
Do we need to consider changing / versioning the cache key?

A user with a valid cache entry for the 'old' SDK version will have `PlatformConfig` in storage, but the 'new' SDK will expect `PlatformConfigAndKeys` from the cache, can it cause any issues?

Reply:
Added V2, the current result is a 4 values in the storage